fe/fe-core/src/main/java/com/starrocks/load/DeleteHandler.java

```diff
@@ -110,6 +112,8 @@
     @SerializedName(value = "dbToDeleteInfos")
     private Map<Long, List<MultiDeleteInfo>> dbToDeleteInfos;
 
+    private ReentrantReadWriteLock lock = new ReentrantReadWriteLock();
```

Comment:
Could you please comment about what is the lock's protection.
`removeOldDeleteInfo` only be called in one thread, why needs a lock to protect?